Dockerfile

```diff
@@ -47,11 +47,24 @@ COPY ./superset-frontend ./
 # This seems to be the most expensive step
 RUN npm run ${BUILD_CMD}
 
+######################################################################
+# Python pre-lean image
```

Comment:
Other options:
- have the release manager bump both `setup.py` and `package.json`, have a CI / build-step that checks that they have been bumped
- have some other mechanism to store the version that get pushed into package.json?
- use a `package.js` insetad of a package.json, that dynamically read some sort of version.txt

I figure this target (pre_lean) could be useful as a place to prepare/cache more python-related stuff eventually. Install more stuff here and COPY into lean